tests/Makefile

```diff
@@ -121,7 +121,8 @@ stop-longevity-test: nfr-test ## Stop the longevity test and collects results
 
 .PHONY: .vm-nfr-test
 .vm-nfr-test: ## Runs the NFR tests on the GCP VM (called by `nfr-test`)
-	go run github.com/onsi/ginkgo/v2/ginkgo --randomize-all --randomize-suites --keep-going --fail-on-pending --trace -r -v --buildvcs --force-newlines $(GITHUB_OUTPUT) \
+	go run github.com/onsi/ginkgo/v2/ginkgo --randomize-all --randomize-suites --keep-going --fail-on-pending --fail-fast \
```

Comment:
I don't think we should necessarily fail fast on NFR or functional tests. Because these tests take awhile to run, if there are multiple failures, we'll only catch one at a time. This will slow down iterations on fixing things.

Reply:
It will only stop running the tests for the suite that had a failure but will keep running the other ones

Reply:
We only have one suite for all these tests. Unless you mean it skips the `Describe` block that had the failure?

Reply:
nope, I mixed them up with unit tests 🙁